src/instructlab/model/train.py

```diff
@@ -87,29 +159,72 @@
         "(reduces GPU VRAM usage and may slow down training)"
     ),
 )
+@click.option("--max-seq-len", type=int)
+@click.option("--max-batch-len", type=int)
```

Comment:
Can we add help text to these args?

Reply:
This is the maximum overall length of samples that we process in a given batch.

Reply:
Great - let's put that in the help text :smile:

Reply:
done.